parsl/usage_tracking/usage.py

```diff
@@ -192,7 +192,17 @@ def send_message(self) -> None:
 
         self.send_UDP_message(message)
 
-    def close(self) -> None:
-        """We terminate (SIGTERM) the processes added to the self.procs list """
+    def close(self, timeout: float = 10.0) -> None:
+        """First give each process one timeout period to finish what it is
+        doing, then kill it (SIGKILL). There's no softer SIGTERM step,
+        because that adds one join period of delay for what is almost
+        definitely either: going to behave broadly the same as to SIGKILL,
+        or won't respond to SIGTERM.
+        """
```

Comment:
From a documentation standpoint, I assume the removal of SIGTERM is less about the join period, and more that shy of a stuck network, it won't do anything.  By the time it could be executed, nominally the child process will have already completed.  Alternatively, the "adds one join period of delay" could be handled by:
```python
for proc in self.procs:
    proc.terminate()
    proc.join(timeout=timeout)
    if proc.is_alive():
        proc.kill()
```
Same timeout with or without the `.terminate()` call.

Meanwhile, I think it is likely correct to remove the `.terminate()`, given how the usage_tracker is utilized during (at least) the shutdown of the DFK in `.cleanup()`.

Reply:
Och; I missed that you said exactly that in the description.  Apologies.